selfdrive/camerad/cameras/camera_common.cc

```diff
@@ -387,6 +387,7 @@ static void driver_cam_auto_exposure(CameraState *c, SubMaster &sm) {
 
 void common_process_driver_camera(SubMaster *sm, PubMaster *pm, CameraState *c, int cnt) {
   if (cnt % 3 == 0) {
+    sm->update(0);
```

Comment:
this doesn't really make sense either unless we restrict the scope of this SubMaster more to the driver cam